packages/docs/src/routes/docs/faq/index.mdx

```diff
@@ -209,3 +210,13 @@ An amazing team of contributors around the world living in [Discord](https://qwi
 ## Is Qwik open source?
 
 Yes, [MIT](https://github.com/BuilderIO/qwik/blob/main/LICENSE) and [dependency-free](https://github.com/BuilderIO/qwik/blob/main/packages/qwik/package.json), installing Qwik will not bloat your node_modules nor your lawyers.
+
+# Does Qwik have any downside?
+
+Yes. Every framework, possessing its own strengths and weaknesses, involves a trade off.
+
+1. As a relative newcomer to the JS world, Qwik has yet to develop a community of libraries and best practices.
+2. Qwik’s magic lies in its serialization, but this also brings about some considerations developers wouldn’t otherwise have to be mindful of.
+3. Where Qwik outperforms most frameworks is on initial page load. If your use case is an SPA, and any level of server side or static rendering is not a need or interest of yours, Qwik may not offer a huge benefit to you
```

Comment:
I would update this to - 

Qwik can load JS apps of any scale - instantly, so its main advantage over current technologies is on initial page load and time to interactive. If your use case is a single page app and you don't mind the time it takes your app to load, adopting Qwik at this stage might not offer you immediate benefits